### PR TITLE
Add contracts, small tests refactor

### PR DESCRIPTION
**Pull request by routegroup**
**Adds:**
- Arrayable contract for Shortcode (toArray method which was previous)
- Test for strip

**Changes:**
- Removes unused imports
- Small refactor for tests

### CODE REVIEW

File: tests/ShortcodesServiceProviderTest.php [hunk 1]
```diff
@@ -1,18 +0,0 @@
...
-{
-    /**
-     * Setup the test environment.
-     */
-    public function setUp()
-    {
-        parent::setUp();
-    }
-
-    public function testShortcodeClass()
```
Comment (routegroup) [inline tests/ShortcodesServiceProviderTest.php]: it was same as testInstance() in tests/ShortcodeTest.php

**Comment by webwizo** [comment]
I am not good in UnitTesting.

**Comment by routegroup** [comment]
@webwizo Thats fine :) It just makes it more readable without trying making one test per file and without duplicating tests.

**Comment by webwizo** [comment]
Thanks!

**Pull request merged**